static/app/views/alerts/list/rules/index.spec.tsx

```diff
@@ -81,7 +88,7 @@ describe('AlertRulesList', () => {
   afterEach(() => {
     act(() => ProjectsStore.reset());
     MockApiClient.clearMockResponses();
-    trackAnalytics.mockClear();
+    jest.restoreAllMocks();
```

Comment:
i think you want clear
```suggestion
    jest.clearAllMocks();
```

Reply:
Idk, the tests still pass lol

Reply:
but yes, clear is more accurate